packages/core-data/src/resolvers.js

```diff
@@ -281,16 +281,10 @@ export const getEntityRecords =
 						.filter( ( record ) => record?.[ key ] )
 						.map( ( record ) => [ kind, name, record[ key ] ] );
 
-					dispatch( {
-						type: 'START_RESOLUTIONS',
-						selectorName: 'getEntityRecord',
-						args: resolutionsArgs,
-					} );
-					dispatch( {
-						type: 'FINISH_RESOLUTIONS',
-						selectorName: 'getEntityRecord',
-						args: resolutionsArgs,
-					} );
+					dispatch.finishResolutions(
+						'getEntityRecord',
+						resolutionsArgs
+					);
```

Comment:
The actions are batched, so we only need `finishResolutions`.

Reply:
Any chance something external would be relying on starting resolution first before marking it as finished?

In other words, what do we lose if we keep both `startResolutions` and `finishResolutions`?

In case you want to go without `startResolutions`, should we add a comment about why we're skipping it?

Reply:
Related https://github.com/WordPress/gutenberg/pull/63430#discussion_r1673992148

Reply:
Thought might be the case. Thanks for confirming 👍